src/content/guides/multi-nginx/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Multi-NGINX ingress controller support
+description: Deploy multiple NGINX ingress controllers in a Kubernetes cluster to separate different ingress traffic classes.
+date: 2020-08-05
+type: page
+tags: ["tutorial"]
+---
+
+# Multi-NGINX ingress controller support
+
+NGINX ingress controller handles [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources, routing the traffic from outside of the Kubernetes cluster to services within the cluster. Starting with App version 1.8.0 one can install multiple NGINX ingress controllers in same Kubernetes cluster. Here we cover common cases where having multiple NGINX ingress controllers can be useful.
+
+Most of the NGINX configuration options have NGINX wide defaults but can also be overriden on per Ingress resource level. In each of the covered multi-NGINX use cases, one essentially installs second NGINX with different NGINX global-only configuration, i.e. configuration that applies to all Ingress resources managed by that NGINX installation which cannot be customized on per-Ingress resource level.
```

Comment:
Most of the NGINX configuration options have NGINX wide defaults but can also be overriden on per Ingress resource level. 
* Would it be helpful to specify what, in general, **can** or **cannot** be overridden? (whatever is fewer)

In each of the covered multi-NGINX use cases, one essentially installs second NGINX with different NGINX global-only configuration, i.e. configuration that applies to all Ingress resources managed by that NGINX installation which cannot be customized on per-Ingress resource level.
* "different NGINX global-only configuration" was confusing to me. But I don't know what to suggest. Is "global-only configuration" a specific term?
* Would sth like, "i.e. configurations managed by an NGINX installation that cannot be customized per-Ingress." be shorter, but clearer?

In each of the covered multi-NGINX use cases, one essentially installs second NGINX with different global NGINX-only configuration, i.e. configuration that applies to all Ingress resources managed by that NGINX installation which cannot be customized on per-Ingress resource level.

Reply:
> Would it be helpful to specify what, in general, can or cannot be overridden? (whatever is fewer)

No, there's extensive upstream nginx configuration docs, and they keep on changing. I've added a link, to both upstream and our chart configuration options.


> "different NGINX global-only configuration" was confusing to me. But I don't know what to suggest. Is "global-only configuration" a specific term?

I made that one up, there's no official term.

> Would sth like, "i.e. configurations managed by an NGINX installation that cannot be customized per-Ingress." be shorter, but clearer?

Maybe it's better, but it's not shorter compared to "NGINX global-only configuration". I've expanded what is meant under that term in the continuation of the sentence.